src/mediatype-icon.ts

```diff
@@ -82,11 +68,16 @@ export class MediatypeIcon extends LitElement {
     }
 
     const displayMediatype = this.displayMediaType || '';
+    const fillColor = mediatypeColor[displayMediatype];
```

Comment:
Question: What happens when `mediatypeColor[displayMediatype]` doesn't exist? When `displayMediatype` is ""?
Does the style attribute get removed?

Reply:
no, it stays there with no value.
same with the icons and icon text